src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -369,9 +383,14 @@
             </Grid.RowDefinitions>
 
             <StackPanel>
-                <Button x:Name="Share" Click="OnShareClick">
-                    <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xE72D;"/>
-	            </Button>
+                <StackPanel Orientation="Horizontal">
```

Comment:
I believe these controls are supposed to be on the left grid over the graph control.

Reply:
Moved.